spanner/crud.js

```diff
@@ -189,10 +233,17 @@ const cli = require(`yargs`)
     {},
     (opts) => readData(opts.instanceName, opts.databaseName)
   )
+  .command(
+    `read-stale <instanceName> <databaseName>`,
+    `Reads data in an example Cloud Spanner table.`,
```

Comment:
Nit: mention/explain stale reads in the command description?

e.g. `Reads _stale_ data in an example Cloud Spanner table.`

Reply:
Done.